src/modules/variable/get.rs

```diff
@@ -124,7 +124,7 @@ impl VariableGet {
                 format!("${name}[{index}]")
             }
             _ => {
-                let index = index.translate_eval(meta, true);
+                let index = index.translate(meta);
```

Comment:
This doesn't seem to make any difference to the test results, so I made it more similar to `slice_copy_array()` below.

Reply:
I assume the eval variant is supposed to do something special, so I don't feel confident changing it to a regular function.

Reply:
I've tried various combinations of by value and by ref argument passing, with and without decrementing the index inside the function, and nothing I do makes any difference to the Bash output.  I've reverted the change anyway, and we can revisit this if it causes problems down the line.